modelci/types/models/mlmodel.py

```diff
@@ -98,11 +110,11 @@ class MLModel(BaseMLModel):
     id: Optional[PydanticObjectId] = Field(default=None, alias='_id')
     parent_model_id: Optional[PydanticObjectId]
     weight: Weight
-    # profile_result: Optional[ProfileResult]
+    profile_result: Optional[Any]
     status: Status = Status.Unknown
     model_status: List[ModelStatus] = Field(default_factory=list)
     creator: str = Field(default_factory=getpass.getuser)
-    create_time: datetime = Field(default_factory=datetime.now, const=True)
+    create_time: datetime = Field(default_factory=datetime.utcnow)
 
 
 class MLModelIn(BaseMLModel):
```

Comment:
In above file, you import MLModelIn from type.models

why does this file also have a MLModelIn?

Reply:
It is like an alias. The actual file is in `types.models.mlmodel`. After import the `MLModelIn` in `types.models.__init__.py`, it can also be import using its alias name `types.models`.

Reply:
so I suggest we make all these `import` consistent otherwise it is confusing for some beginners to join our contributor team.

Reply:
Done